uw-frame-components/portal/misc/partials/app-header.html

```diff
@@ -1,6 +1,9 @@
 <md-subheader role="heading" class="app-header">
   <h1 flex="50" flex-xs="85" class="app-title">
-    <i class="fa {{::icon}}" aria-hidden="true"></i> {{::title}}</h1>
+    <i ng-if="icon.indexOf('fa-') > -1" class="fa {{::icon}}" aria-hidden="true"></i>
+    <md-icon ng-if="icon.indexOf('fa-') <= -1">{{ ::icon }}</md-icon>
```

Comment:
I think this [needs an aria-label](https://material.angularjs.org/latest/api/directive/mdIcon). Maybe an aria-label of `""` to get the `aria-hidden="true"` behavior.